catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -207,29 +207,29 @@
             Metacard.METADATA,
             true /* indexed */,
             true /* stored */,
-            false /* tokenized */,
```

Comment:
Why does BasicTypes have duplicate descriptors instead of just referencing the descriptors in the *Attributes classes?